CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
-# v1.0.0
+# 1.1.0
```

Comment:
It's a fix to make this better approximate LS 7+ behaviour on LS6. Let's bump just the patch level.